package.json

```diff
@@ -21,13 +21,12 @@
   },
   "devDependencies": {
     "babel": "^5.5.5",
-    "babel-eslint": "^3.1.14",
-    "babel-loader": "^5.1.4",
+    "babel-plugin-object-assign": "^1.1.0",
     "browserify": "^10.2.4"
   },
   "scripts": {
-    "prepublish": "node_modules/babel/bin/babel/index.js src/main.jsx -o lib/main.js",
```

Comment:
no need to go into node_modules in npm. It automatically searches for binaries in there. so just `babel` is enough.